### PR TITLE
Add a line under profile image when editing profile

### LINKED ISSUE #3136: Add a line under profile image when editing profile
### Description

*Type*: Bug

*Summary*: Add a line to profile for both platforms to match the mock

#### Expected behavior

image

iOS 
image

Android
image

#### Actual behavior
[comment]: # (Describe what actually happened.)

### Reproduction
- Open Status
- Tap Profile
- Tap edit
- Pay attention to the user profile

### PR DESCRIPTION
**Pull request by salax**
fixes #3136

### Summary: Add a line under profile image when editing profile

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/profile/styles.cljs [hunk 1]
```diff
@@ -44,6 +44,13 @@
    :justify-content :center
    :align-items     :center})
 
+(def profile-badge-edit
+  {:flex-direction  :column
+   :justify-content :center})
+
+(def modal-menu-center
```
Comment (goranjovic) [inline src/status_im/ui/screens/profile/styles.cljs]: Please rename to `modal-menu` - style names should only reflect what they refer to, not how they style it.

**Comment by asemiankevich** [comment]
Branch: PR-3148
Android: https://i.diawi.com/YwJgYW
iOS: https://i.diawi.com/7jvBMy


Android:
image

iOS
image

**Comment by jeluard** [comment]
@salax It would be nicer semantically to have an extra `view` used as separator (as https://github.com/status-im/status-react/blob/develop/src/status_im/ui/screens/wallet/components/views.cljs#L161). WDYT?

<blockquote><div> GitHub</div><div><strong><a href="https://github.com/status-im/status-react">status-im/status-react</a></strong></div><div>status-react - a free (libre) open source, mobile OS for Ethereum</div></blockquote>

**Comment by jeluard** [comment]
@salax My bad, this is only in edit mode. Your solution looks right then.

**Review by jeluard — APPROVED** [review]

**Review by goranjovic — APPROVED** [review]

**Comment by andytudhope** [comment]
@salax thank you and congratulations on your first PR. Could you please squash the commits here and then I will merge and confirm the bounty for you.

**Comment by salax** [comment]
done squashing, but not sure why build failed...

**Comment by goranjovic** [comment]
@salax You might want to rebase against develop branch. Let's see if that fixes the build

**Comment by salax** [comment]
Tried that, but it didn't work. Looks like last commit in development failed to build too https://github.com/status-im/status-react/commit/c4151534fb35ebcc95d270b2fbcfa25d2f5a9760

**Comment by salax** [comment]
interesting it passed this time...

**Comment by goranjovic** [comment]
Just ran it one more time, it works fine now. There was some issue with the CI, but that's fixed.  
In any case, your PR tested ok and is in the queue to be merged as soon as possible.

Once again, thanks for your contribution and good work! :)

**Comment by pablanopete** [comment]
@salax Thanks so much for contributing to Status through Status open bounty; when you have a chance could you drop by chat.status.im and give me a ping (hutch)!

**Pull request merged**